utils/github/getJsonData.ts

```diff
@@ -9,13 +9,19 @@ const getJsonData = async (
   sourceProviderConnection: SourceProviderConnection
 ) => {
   if (sourceProviderConnection) {
+
     const response = await getContent(
       sourceProviderConnection.forkFullName,
       sourceProviderConnection.headBranch || 'master',
       filePath,
       sourceProviderConnection.accessToken
     )
 
+    if (response.response) {
+      if (response.response.status == 404)
+        return 'Content not found. Your fork may have been deleted.'
```

Comment:
I think the build is failing because this isn't always expected to be a string. Maybe we should throw an error here instead (or return an object with a error prop). We'll have to do the same thing with the getMarkdownData below as well